### PR TITLE
Fix a couple of bugs

### PR DESCRIPTION
**Pull request by guotuofeng**
Fix bug https://github.com/pytorch/kineto/issues/130
https://github.com/pytorch/kineto/issues/270
https://github.com/pytorch/kineto/issues/338

### CODE REVIEW

**Comment by guotuofeng** [comment]
image

File: tb_plugin/fe/src/app.tsx [hunk 1]
```diff
@@ -153,8 +162,10 @@ export const App = () => {
       try {
         const runs = await api.defaultApi.runsGet()
         setRuns(runs)
+        setHasRuns(true)
```
Comment (TomWildenhain-Microsoft) [inline tb_plugin/fe/src/app.tsx:165]: Won't this set `hasRuns` to `true` even if the runs returned from the backend is empty?

Comment (TomWildenhain-Microsoft) [reply tb_plugin/fe/src/app.tsx:165]: I don't think you need a new state variable.  Can you just check `if (runs)` in any place you would do `if (hasRuns)`?

Comment (guotuofeng) [reply tb_plugin/fe/src/app.tsx:165]: I need the hasRun to differentiate whether the load is done but no any run or the loading is in progress

File: tb_plugin/fe/src/app.tsx [hunk 2]
```diff
@@ -248,6 +259,22 @@ export const App = () => {
   }
 
   const renderContent = () => {
+    if (!hasRuns) {
+      return (<Card className={classes.container}>
```
Comment (TomWildenhain-Microsoft) [inline tb_plugin/fe/src/app.tsx:263]: This will be more consistent with the rest of the UI if you use CardHeader.

File: tb_plugin/fe/src/app.tsx [hunk 3]
```diff
@@ -122,12 +125,18 @@ const useStyles = makeStyles((theme) => ({
   },
   iconButton: {
     padding: '8px'
-  }
+  },
+  container: {
+    display: 'flex',
+    padding: '80px',
```
Comment (TomWildenhain-Microsoft) [inline tb_plugin/fe/src/app.tsx:131]: Any reason for the enormous padding?  If you want the text closer to the center of the screen, I'd prefer a margin so the card to text distance is consistent with other cards.

**Review by nbcsm — APPROVED** [review]

File: tb_plugin/fe/src/app.tsx (hunk 1 above)
Comment (TomWildenhain-Microsoft) [reply tb_plugin/fe/src/app.tsx:165]: Ah, I see.  The backend returns a 404, not an empty list when it finishes scanning the directory if there are no runs.

**Review by TomWildenhain-Microsoft — CHANGES_REQUESTED** [review]

**Pull request closed without merging**